common/tqid/task_queue_validator_test.go

```diff
@@ -0,0 +1,192 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
For new files let's update the year and remove the Uber line. I think you're also missing the license header from `task_queue_validator`